ngraph/core/src/op/loop.cpp

```diff
@@ -185,12 +185,9 @@ void op::v5::Loop::validate_and_infer_types()
                           "Number of inputs must be the same as number of input descriptions");
 
     // Input
-    uint64_t index_it = input_offset;
```

Comment:
Can we create some test for this? May be add a IE Reader test (where XML file is read)

Reply:
@GlebKazantaev @ilyachur Do we have ngraph_reader unit tests in pre-commits?
TEST_F(NGraphReaderTests, ReadLoopNetwork)  already reproduces the issue

>   Actual: it throws:Check 'index == index_it' failed at core/src/op/loop.cpp:192:
While validating node 'v5::Loop Loop_85 (trip_count/Output_0/Data__const[0]:i64{1}, a_final5/execution_cond/Output_0/Data__const[0]:boolean{}, a_init[0]:f32{1,2}) -> ()' with friendly_name 'Loop_85':
Input_index not in order

Reply:
the test has been added